Extension/src/LanguageServer/client.ts

```diff
@@ -3439,10 +3449,55 @@ export class DefaultClient implements Client {
             }
         }
 
-        if (uri === undefined || range === undefined) {
+        if (uri === undefined || range === undefined || editor === undefined) {
             return;
         }
 
+        let formatParams;
+        if (cppSettings.useVcFormat(editor.document))
+        {
+            const editorConfigSettings: any = getEditorConfigSettings(uri.fsPath);
+            formatParams = {
+                editorConfigSettings: {},
```

Comment:
Shouldn't `editorConfigSettings` be populated here?

Reply:
it is, I tested this and it works.